File: HW/work.cpp
#include <iostream>
#include <cstdio>
#include <cstring>
#include "work.h"

using std::cin;
using std::cout;
using std::endl;
using std::string;

Work::Work(string &myfirstName, string &mylastName, int &myphoneNumber, string &myteam) : Person(myfirstName, mylastName, myphoneNumber) 
{
    team = myteam;
}

void Work::setTeam(string myteam)
{
    team = myteam;
}

string Work::getTeam()
{
    return team;
}

void Work::print()
{
    cout << getFirstName() << " " << getLastName() << "_";
    int phonenum = getPhoneNumber();
    int_to_string(phonenum);
    cout << "_" << getTeam() << endl;
}

File: HW_java/Work.java
public class Work extends Person
{
    private String team;
    
    public Work(String myfirstName, String mylastName, int myphoneNumber, String myteam)
    {
	super(myfirstName, mylastName, myphoneNumber);
	team = myteam;
    }

    public void setTeam(String myteam)
    {
	team = myteam;
    }

    public String getTeam()
    {
	return team;
    }

    public void print()
    {
	System.out.print(getFirstName() + " " + getLastName() + "_");
	int phonenum = getPhoneNumber();
	int_to_string(phonenum);
	System.out.println("_" + getTeam());
    }
};    
